module1/homework/removeVowels/vowel.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <algorithm>
+#include <array>
+#include <vector>
+
+inline constexpr int vowels_number=17;
```

Comment:
In fact, it's 12. The rest of the array will be filled with char with 0 code. It's not something that we wanted to remove.